test/Rln.t.sol

```diff
@@ -110,6 +113,7 @@ contract RlnTest is Test {
     }
 
     function test__InvalidSlash__InvalidIdCommitment(uint256 idCommitment) public {
+        vm.assume(rln.isValidCommitment(idCommitment));
```

Comment:
This looks like we want to update the test signature. Unless I'm misunderstanding, we're ensuring here that `idCommitment` is always valid. So we will never have a case of `InvalidIdCommitment`.

Should we add a test that checks for `InvalidIdCommitment` revert reason explicitly?

Reply:
Addressed in [4cc2a27](https://github.com/vacp2p/rln-contract/pull/26/commits/4cc2a27db03338f02b24d7e695ffbe8d0a38cc06)

Reply:
Added the test here - https://github.com/vacp2p/rln-contract/pull/26/commits/077be3200ccca2cc4fc06422c9a66489e8fa1881